plugins/modules/zos_operator.py

```diff
@@ -32,16 +32,37 @@
     required: true
   verbose:
     description:
-      - Return verbose information.
+      - Return diagnostic messages that lists and describes the execution of the
+        operator commands.
+      - Return security trace messages that help you understand and diagnose the
+        execution of the operator commands
+      - Return trace instructions displaying how the the command's operation is
+        read, evaluated and executed.
     type: bool
     required: false
     default: false
-  debug:
+  wait_time_s:
```

Comment:
@richp405 this is not a show stopper, maybe a max check here of 2147483647 or something, what is going to happen if someone enters 21474836471234 beyond the 2^32?

Reply:
@blakeinate thanks for reminding in python 3 int's are unbounded :) , so @richp405 we can consider this comment moot.